src/services/book.service.js

```diff
@@ -1,11 +1,25 @@
 import angular from 'angular';
-import books from './data-books.json';
 
-angular.module('app-bootstrap').factory('bookService', [
-  function() {
+angular.module('app-bootstrap').factory('bookService', ['$http', 'localStorageService',
+  function($http, localStorageService) {
+    const accessToken = localStorageService.get('access-token');
+    const client = localStorageService.get('client');
+    const uid = localStorageService.get('uid');
     return {
-      getBooks: () => books,
-      getBook: (id) => books.find((book) => book.id === parseInt(id))
+      getBooks: () => $http.get(`${process.env.API_URL}/books`, {
+        headers: {
+          'access-token': accessToken,
+          client,
+          uid
+        }
```

Comment:
I think you can create a variable or something else so you won't have to repeat this on every request 🤔